doc/syscalls/README.md

```diff
@@ -125,4 +125,3 @@ _Note:_ GPIO is slated for re-numbering in Tock 2.0.
 |   | 0x80002       | PCA9544A         | I2C address multiplexing                   |
 |   | 0x80003       | GPIO Async       | Asynchronous GPIO pins                     |
 |   | 0x80004       | nRF51822         | nRF serialization link to nRF51822 BLE SoC |
-|   | 0x80005       | [HD44780](80005_hd44780.md)          | LCD HD44780 capsule                        |
```

Comment:
Can you add the text screen to this doc somewhere?

Reply:
I added the documentation for the touch panel and the two screen drivers.